admin_manual/configuration_server/hardening.rst

```diff
@@ -10,6 +10,15 @@ This document lists some security hardenings which require manual interaction by
           however in some cases that are considered second level as defense administrators are encouraged to check these
           hardenings manually.
 
+Deployment
+----------
+
+Move data directory outside of the web root
+*******************************************
+It is highly recommended to move the data directory (where ownCloud stores its data) outside of the web root (i.e. outside
+of ``/var/www``), this can be done by modifying the ``datadirectory`` switch in the configuration file. It is possible to
+do this also after an instance has been installed by moving the folder manually.
```

Comment:
> But IIRC there where also some ML postings in the past which states that a datadirectory shouldn't be moved. If thats not true anymore for Linux-based systems then i'm fine. :+1:

It should work fine if you use ownCloud 8 as the legacy storages got automatically converted. At least they should.

@PVince81 Care to verify? :)